lib/main.js

```diff
@@ -163,6 +238,10 @@ function loadLibraries() {
                                      [ctypes.char.ptr,
                                       ctypes.int,
                                       ctypes.voidptr_t]);
+  PK11_CheckUserPassword = declareFunction("PK11_CheckUserPassword", nss3,
+                                     [SECStatus,
```

Comment:
The alignment I've been trying to stick to would put the start of this line just after the '(' on the previous line (although you can see I wasn't consistent for SEC_PKCS7Encode).